### PR TITLE
Prevent unnecessary sensor iterations in `updateSensorNode()`

### PR DESCRIPTION
**Pull request by SwoopX**
By moving cluster related code to dedicated files, those clusters have been removed in the whitelist of `updateSensorNode()`. However, the clusters are just checked while the sensor iteration has already entered and consequently, all sensors are looped for a non-regarded cluster.

The check for valid clusters is now moved up at the very beginning, before the sensor loop, thereby potentially sparing quite a number of unnecessary loops.

### CODE REVIEW

**Comment by Smanar** [comment]
This part is no more needed ?

```
               // ubisys device management (UBISYS_DEVICE_SETUP_CLUSTER_ID)
                if (event.endpoint() == 0xE8 && existDevicesWithVendorCodeForMacPrefix(event.node()->address(), VENDOR_UBISYS))
                {
                    break;
                }
```

**Comment by SwoopX** [comment]
No, because there never was any processing for it, so it's unnecessary code.

File: de_web_plugin.cpp [hunk 1]
```diff
@@ -8101,57 +8128,15 @@ void DeRestPluginPrivate::updateSensorNode(const deCONZ::NodeEvent &event)
             return;
         }
 
-        // filter for relevant clusters
-        if (event.profileId() == HA_PROFILE_ID || event.profileId() == ZLL_PROFILE_ID)
+        if (event.clusterId() == VENDOR_CLUSTER_ID && !(i->modelId() == QLatin1String("de_spect") && existDevicesWithVendorCodeForMacPrefix(event.node()->address(), VENDOR_DDEL)))
```
Comment (manup) [inline de_web_plugin.cpp:8131]: The `"de_spect"` check can be removed. This was only a internal test device.

Comment (SwoopX) [reply de_web_plugin.cpp]: You got it, done.

**Review by manup — APPROVED** [review]

**Pull request merged**